### ISSUE TITLE
Add Docker build to Travis CI

### ISSUE THREAD

**Issue by vietlq** [Consensys-Incorporated/quorum#590]
I can see that current Travis CI config does not have a step to build Docker images:

https://travis-ci.org/jpmorganchase/quorum/jobs/463802953

Adding Docker build to Travis CI will guarantee that a PR will not break Docker build. Since Travis runs builds in parallel, only some extra time will be needed due to the fact that Docker is slightly slower.

Adding Docker step will be easy: https://docs.travis-ci.com/user/docker/

If there are no objections, I can create a new PR.

**Comment by trung** [comment]
We use Docker Cloud automated pipeline to build our images and would like to make Travis build faster for fast feedbacks.

Docker Cloud offers tag build by configuration (vs handcrafted script in travis yaml) which is important during the release.

**Comment by vietlq** [comment]
Thanks :)
